src/sass/components/navbar.module.scss

```diff
@@ -56,4 +56,122 @@
         color: $accent;
     }
     
-}
\ No newline at end of file
+    
+}
+@media screen and (min-width: 751px){
+    .navHamburger{
+        display: none;
+    }
+}
+@media screen and (min-width: 600px) and (max-width: 750px){
+    .resInp{
+        display: block;
+        width: 25px;
+        height: 25px;
+        position: absolute;
+        top: 5px;
+        right: 5em;
+        /* left: -5px; */
+        cursor: pointer;
+        opacity: 0;
+        z-index: 2; /* and place it over the hamburger */
+        -webkit-touch-callout: none;
+    }
+}
+@media screen and (max-width: 750px) {
+    .navList, .navIcon p{
+        display: none;
+    }
+    .navHamburger{
+        display: flex;
+        justify-content: center;
+        align-items: center;
+    }
+    .navHeader h1{
+        font-size:large;
+    }
+   
+    .hamSpan{
+        display: block;
+        width: 22px;
+        height: 4px;
+        margin-bottom: 5px;
+        position: relative;
+        background: white;
+        border-radius: 3px;
+        z-index: 1;
+        transform-origin: 4px 0px;
+        transition: transform 0.5s cubic-bezier(0.77,0.2,0.05,1.0),background 0.5s cubic-bezier(0.77,0.2,0.05,1.0), opacity 0.55s ease;
+    }
+
+    .hamSpan:first-child{
+        transform-origin: 0% 0%;
+    }
+
+    .hamSpan:nth-last-child(2){
+        transform-origin: 0% 100%;
+    }
+
+    .resInp:checked ~ .hamSpan{
+        opacity: 1;
+        transform: rotate(45deg) translate(2px, 5px);
+        background: white;
+    }
+    .resInp:checked ~ .hamSpan:nth-last-child(3){
+        opacity: 0;
+        transform: rotate(0deg) scale(0.2, 0.2);
+    }
+
+    .resInp:checked ~ .hamSpan:nth-last-child(2){
+        transform: rotate(-45deg) translate(0, -1px);
+    }
+ 
+    .hamburger{
+        /* display: none; */
+        position: absolute;
+        width: 100%;
+        margin: -100px 0 0 -86%;
```

Comment:
change the margin of this line to this because the hamburger => margin: -100px 0 0 -91%;